src/controller.cpp

```diff
@@ -21,6 +21,21 @@ namespace {
 constexpr double RADIUS_EPS = 0.001; // Smallest relevant radius [m]
 constexpr double LONG_DURATION = 31556926; // A year (ros::Duration cannot be inf)
 
+// Cutoff frequency for the derivative calculation in Hz.
+constexpr double cutoff_frequency_lat = -1.0;
+constexpr double cutoff_frequency_ang = -1.0;
+
+// Upper and lower saturation limits
+constexpr double lat_upper_limit = 100.0;
+constexpr double lat_lower_limit = -100.0;
+
+constexpr double ang_upper_limit = 100.0;
+constexpr double ang_lower_limit = -100.0;
+
+// Anti-windup term. Limits the absolute value of the integral term.
+constexpr double windup_limit = 1000.0;
+
+
```

Comment:
@cesar-lopez-mar are these parameters the same for all robots, or should they be user configurable?

Reply:
I created a more generic issue for this question in #35